src/integration/kotlin/com/coditory/gradle/integration/ConfigurationInheritanceTest.kt

```diff
@@ -0,0 +1,92 @@
+package com.coditory.gradle.integration
+
+import com.coditory.gradle.integration.base.GradleTestVersions.GRADLE_MAX_SUPPORTED_VERSION
+import com.coditory.gradle.integration.base.GradleTestVersions.GRADLE_MIN_SUPPORTED_VERSION
+import com.coditory.gradle.integration.base.TestProjectBuilder
+import org.assertj.core.api.Assertions.assertThat
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.AutoClose
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+class ConfigurationInheritanceTest {
+    companion object {
+        @AutoClose
+        private val project = TestProjectBuilder
+            .project("project-${ConfigurationInheritanceTest::class.simpleName}")
+            .withBuildGradleKts(
+                """
+                plugins {
+                    id("com.coditory.integration-test")
+                }
+
+                repositories {
+                    mavenCentral()
+                }
+
+                dependencies {
+                    testImplementation("org.junit.jupiter:junit-jupiter-api:${Versions.junit}")
+                    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${Versions.junit}")
+                }
+
+                testing {
+                    suites {
+                        register<JvmTestSuite>("customTest")
+                    }
+                }
+
+                val customTestImplementation by configurations.getting {
+                    extendsFrom(configurations.integrationImplementation.get())
+                }
+                val customTestRuntimeOnly by configurations.getting {
+
+                    extendsFrom(configurations.integrationRuntimeOnly.get())
+                }
+
+                dependencies {
+                    testImplementation("org.junit.jupiter:junit-jupiter:${Versions.junit}")
+                    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
Could you change it to `testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${Versions.junit}")` and drop the `dependencies { ... }` from the top?

Reply:
Fixed! Idk how two dependencies block ended in PR 😅